servers/xgboostserver/Makefile

```diff
@@ -1,6 +1,6 @@
-VERSION=0.3
+VERSION=0.4
 IMAGE_BASE=seldonio/xgboostserver
-S2I_IMAGE_VERSION=0.18
+S2I_IMAGE_VERSION=1.1.1-SNAPSHOT
```

Comment:
The s2i image should be read from the version.txt file in the top level repo

Reply:
Hmm... that's not the case for other prepackaged model servers... 
Also if `version.txt` will be bumped but we won't rebuild prepackaged model server it won't be clear at glance with what version of s2i image server was built...

We could however bump all versions of prepackaged model servers to follow `version.txt` from top level.

Reply:
Good point! Hmm yeah I see what you mean, then in that case it's the question of whether we'd want to release a version of the prepackaged model server every time. Having said that, if these servers are supposed to be GA then that would make sense. Would be good to hear thoughts from @gsunner  & @cliveseldon

Reply:
Leaving here as it is and will followup in separate issue https://github.com/SeldonIO/seldon-core/issues/1726.